Program.cs

```diff
@@ -118,7 +120,11 @@ static void Main(string[] args)
                 }
 
                 var snapshotTipHeader = snapshotTipDigest.Value.Header;
-                var json = JsonSerializer.Serialize(snapshotTipHeader);
+                var jObj = new JObject();
+                jObj = JObject.FromObject(snapshotTipHeader);
+                jObj.Add("APV", apv);
+                var json = JsonConvert.SerializeObject(jObj);
```

Comment:
How about naming like `jsonObject`? It is just a suggestion and there is no strict rule related to this. So, if you don't agree with this, you can skip this suggestion 😉 

```suggestion
                var jsonObject = new JObject();
                jsonObject = JObject.FromObject(snapshotTipHeader);
                jsonObject.Add("APV", apv);
                var json = JsonConvert.SerializeObject(jsonObject);
```

Reply:
I agree! I wasn't entirely sure about variable naming so I appreciate the suggestion! 🙏